gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemNewIntegrationTest.java

```diff
@@ -1027,8 +1027,13 @@ public void rename_file_sequential() throws Exception {
                 bucketName, dirObject + "/", /* maxResults= */ 1, /* pageToken= */ null),
             getRequestString(bucketName, dirObject + "/"),
             copyRequestString(
-                bucketName, dirObject + "/f1", bucketName, dirObject + "/f2", "copyTo"),
-            deleteRequestString(bucketName, dirObject + "/f1", 1));
+                bucketName,
+                dirObject + "/f1",
+                bucketName,
+                dirObject + "/f2",
+                "copyTo",
+                /* generationId= */ 1),
+            deleteRequestString(bucketName, dirObject + "/f1", 2));
```

Comment:
Why is the generation id changed to 2 for source?
We didn't update /f1, is it because we ran a copy operation on it?

Reply:
We mock increment genId, so any new check on genId, we would need to modify (increment) the rest in the assertion